pkg/reconciler/pipelinerun/pipelinerun_test.go

```diff
@@ -3344,19 +3345,35 @@ func TestReconcileWithTimeoutAndRetry(t *testing.T) {
 	for _, tc := range []struct {
 		name               string
 		retries            int
+		retriesStatus      string
 		conditionSucceeded corev1.ConditionStatus
 		wantEvents         []string
 	}{{
-		name:               "One try has to be done",
-		retries:            1,
+		name:    "One try has to be done",
```

Comment:
When I've encountered these tests before I've really struggled to understand what they do, and I think now they might even be a bit more confusing, since if retries are a taskrun feature why test them on a PipelineRun? Could you update the test comment and test names to clarify? I'm also not sure why `conditionSucceeded` is changing in the second test case.